platform/event/event_payloads.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2024 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import "event_payload.proto";
+
+option java_package = "com.swirlds.proto.legacy.event";
+// <<<pbj.java_package = "com.swirlds.proto.event">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * A list payloads of an event
+ */
+message EventPayloads {
```

Comment:
I'm curious what this is used for. We could save some bytes by directly using `repeated EventPayload payloads = 1;` in a message instead of the wrapper `EventPayloads`.

Reply:
We need it for payload retraction. The hash of an event will consist of the hash of all payloads plus other event data. In order to get that hash independently, I think we need to have the payloads as a separate message, unless I'm missing something.